recipes-extended/onie-tools/onie-tools/onie-bisdn-upgrade

```diff
@@ -54,7 +56,10 @@ case $# in
 esac;
 
 if [ "${NEED_CONFIRM}" = true ]; then
-  read -p "Upgrading BISDN Linux to install-source: ${INSTALL_FILE} using Client-IP-cfg: dhcp! Please confirm system reboot: [y/N] " yn
+  echo "Upgrading BISDN Linux from ${INSTALL_FILE}."
+  echo "ONLY FILES CONFIGURED FOR RETENTION WILL BE KEPT!"
+  echo "For additional information, see the man page for onie-bisdn-upgrade(1)."
```

Comment:
Maybe we should be there a bit more screaming like ssh does for changed ssh host keys:

```suggestion
  echo "Upgrading BISDN Linux from ${INSTALL_FILE}."
  echo ""
  echo "############################################################################"
  echo "# ONLY FILES CONFIGURED FOR RETENTION WILL BE KEPT!                        #"
  echo "#                                                                          #"
  echo "# For additional information, see the man page for onie-bisdn-upgrade(1).  #"
  echo "############################################################################"
  echo ""
```

Reply:
Done.